app/test/frontend/golden/pkg_show_page.html

```diff
@@ -333,8 +333,7 @@ <h3 class="title">About</h3>
       <p>my package description</p>
       <p>
           <a class="link" href="http://hans.juergen.com">Homepage</a><br/>
-          
-          <a class="link" href="/documentation/foobar_pkg/latest/">API Docs</a>
+          <a class="link" href="/documentation/foobar_pkg/latest/">API Docs</a><br/>
```

Comment:
This adds a trailing `<br/>`. Is that of consequence?

Reply:
Yeah, also it removes an extra newline. Its from the template change: now the loop adds a <br/> even though it may be the last one. The rendered version is the same, so I kept the template simple and just accepted the extra <br/>